src/utils/isStandardSyntaxMediaFeature.js

```diff
@@ -16,5 +16,8 @@ export default function (mediaFeature) {
   // SCSS or Less interpolation
   if (hasInterpolation(mediaFeature)) { return false }
 
+  // SCSS variables
+  if (/\$.+?/.test(mediaFeature)) { return false }
```

Comment:
@evilebottnawi The regex `/$.+?/` might be too restrictive. It only checks that we have a `$` anywhere in the string followed by anything. It outlaws SCSS variables even if there is no operator involved.